fdbbackup/backup.actor.cpp

```diff
@@ -2345,12 +2353,28 @@ ACTOR Future<Void> listDBMove(Database src, Database dest) {
 ACTOR Future<Void> finishDBMove(Database src, Key srcPrefix, Optional<double> maxLagSeconds) {
 	try {
 		// Send request to source cluster
-		state FinishSourceMovementRequest finishSourceMovementRequest(srcPrefix, maxLagSeconds.orDefault(DBL_MAX));
+		state FinishSourceMovementRequest finishSourceMovementRequest(srcPrefix, maxLagSeconds);
 		state Future<ErrorOr<FinishSourceMovementReply>> finishSourceMovementReply = Never();
 		state Future<Void> initialize = Void();
+		state double timeoutLimit;
+		if (maxLagSeconds.present()) {
+			timeoutLimit = maxLagSeconds.get();
+		} else {
+			state TenantMovementStatus status = wait(getMovementStatus(src, srcPrefix, MovementLocation::SOURCE));
+			if (!status.mutationLag.present()) {
+				fprintf(stderr,
+				        "The movement with prefix: %s on database: %s is not ready to be finished.\n",
+				        srcPrefix.toString().c_str(),
+				        src->getConnectionRecord()->getConnectionString().toString().c_str());
```

Comment:
I think we can omit the details since they were just passed in.

```suggestion
				fprintf(stderr, "ERROR: The movement is not ready to be finished.\n");
```